nuxeo-features/nuxeo-automation/nuxeo-automation-io/src/main/java/org/nuxeo/ecm/automation/jaxrs/io/documents/JsonESDocumentWriter.java

```diff
@@ -245,18 +246,17 @@ protected static void writeProperties(JsonGenerator jg, DocumentModel doc, Strin
         if (prefix == null || prefix.length() == 0) {
             prefix = schema;
         }
-        prefix = prefix + ":";
+        JSONPropertyWriter currentWriter = writer.prefix(prefix);
```

Comment:
Given that you have to specify a prefix which creates a new `JSONPropertyWriter` object you should just ditch the `static` (rarely a good idea) and just create a new `JSONPropertyWriter` here and don't bother to make the fluent methods return new instances.